examples/tv-casting-app/darwin/MatterTvCastingBridge/MatterTvCastingBridge/templates/MCCommandPayloads.zapt

```diff
@@ -0,0 +1,51 @@
+{{> header excludeZapComment=false}}
+
+#import "MCInteractionModelStructs.h"
+
+#import <Matter/MTRDefines.h>
+#import <Matter/MTRStructsObjc.h>
```

Comment:
Do we really need these? CI failure: https://github.com/project-chip/connectedhomeip/actions/runs/8335319917/job/22810591750?pr=32626

There is a build failure due to them:

```
1751#import <Matter/MTRDefines.h>
1752 ^~~~~~~~~~~~~~~~~~~~~
1753/Users/runner/work/connectedhomeip/connectedhomeip/examples/tv-casting-app/darwin/MatterTvCastingBridge/MatterTvCastingBridge/zap-generated/MCCommandPayloads.h:22:9: note: did not find header 'MTRDefines.h' in framework 'Matter'
```